### PR TITLE
Project week 6/Atefeh-Ulysses-Nazanin

### PR DESCRIPTION
**Pull request by nazi-pira**
Link to my deployed project:
https://priceless-chandrasekhar-e94c89.netlify.app/

My reflections on how this week's project turned out:
This week's project has cheered us up by how we can make a practical application after just six week in the bootcamp! We have faced problems regarding the Date and the min & max temperature but has managed to figure a solution out by some googling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=width-device, initial-scale=1.0">
```
Comment (Ingetora) [inline index.html]: We get an warning in console for width-device. But it seems it's not affecting the app.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    <div>${Math.round(item.main.temp_min)}</div>
+                    <img src=${iconSrc} />
+                </div>
+
+            `
+        })
+
+    })
+    
+
```
Comment (Ingetora) [inline script.js]: Very nice, clean code. Easy to understand. App is fully functional and straight forward to use. Hamburger menu doesn't seem to have a function, and should mabye be removed.

**Review by Ingetora — COMMENTED** [review]
Nice job! Functional app, we would use it! 

// Vlad and Maria

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Your app has very nice logic and functionality 🦄 
You need to however work on styling a little more. Responsiveness is not to create a cut-out small area from the background, but make your app look good on every screen. 
You are very close to meet the requirements for this project! 🦾

**Comment by nazi-pira** [comment]
> Your app has very nice logic and functionality 🦄
> You need to however work on styling a little more. Responsiveness is not to create a cut-out small area from the background, but make your app look good on every screen.
> You are very close to meet the requirements for this project! 🦾

Thank you so much for the nice comments! we would be appreciated if you enlighten us with a hint or a suggestion of the needed changes.

**Review by KarolCislowski — APPROVED** [review]

**Pull request closed without merging**